frontend/src/components/VideoStream/VideoSection.tsx

```diff
@@ -0,0 +1,37 @@
+import { Button, Card, Icon } from '@equinor/eds-core-react'
+import { fullscreen } from '@equinor/eds-icons'
+import styled from 'styled-components'
+
+import { VideoStream } from 'models/VideoStream'
+
+interface VideoSectionProps {
+    videoStream: VideoStream
+    toggleFullScreenModeFunction: VoidFunction
+}
+
+const StyledVideoSection = styled(Card)`
+    height: 95%;
+    display: inline-flex;
+`
+
+const FullscreenButton = styled(Button)`
+    position: absolute;
+    bottom: 8px;
+    right: 8px;
+    width: 32px;
+    height: 32px;
+    opacity: 0.75;
+`
+
+Icon.add({ fullscreen })
+
+export function VideoSection({ videoStream, toggleFullScreenModeFunction }: VideoSectionProps) {
+    return (
+        <StyledVideoSection>
+            <img src={videoStream.url} width="100%" height="100%" alt="robot video stream" />
```

Comment:
Maybe include the `videostream.name` in the `alt` property?

Reply:
Agreed